app/assets/stylesheets/application.scss

```diff
@@ -24,17 +24,42 @@ body {
   background-size: cover;
 }
 
+footer {
+  height: auto;
+  background-color: black;
+}
+
+.banner-image {
+  padding-top: 15px;
+}
+
+.section-title {
+  font-family: 'Miltonian Tattoo', cursive;
+}
+
+.section-box {
+  background-color: gray;
+  padding: 10px;
+  border: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.